src/DataTypes/DataTypeTuple.cpp

```diff
@@ -156,8 +156,15 @@ MutableColumnPtr DataTypeTuple::createColumn() const
 
 MutableColumnPtr DataTypeTuple::createColumn(const ISerialization & serialization) const
 {
-    const auto & element_serializations =
-        assert_cast<const SerializationTuple &>(serialization).getElementsSerializations();
+    const auto * current_serialization = &serialization;
```

Comment:
There are no comments.
I opened `SerializationNamed.h` and there are no comments either.